packages/@aws-cdk/aws-ssm/lib/parameter.ts

```diff
@@ -333,6 +333,23 @@ export interface SecureStringParameterAttributes extends CommonStringParameterAt
  * @example
  *
  * const ssmParameter = new ssm.StringParameter(this, 'mySsmParameter', {
+<<<<<<< HEAD
```

Comment:
Looks like the merge went wrong here.

Reply:
Sorry about that! I must have pulled from main on my local fork.